files/en-us/learn/html/cheatsheet/index.html

```diff
@@ -171,9 +171,12 @@ <h2 id="Inline_elements">Inline elements</h2>
   </tr>
   <tr>
    <td>Embedded Video</td>
-   <td>&lt;video controls src="https://archive.org/download/WebmVp8Vorbis/webmvp8_512kb.mp4"&gt;The &lt;code&gt;video&lt;/code&gt; element is unsupported.&lt;/video&gt;</td>
+   <td>&lt;video controls src="https://interactive-examples.mdn.mozilla.net/media/cc0-videos/flower.webm"&gt;The &lt;code&gt;video&lt;/code&gt; element is unsupported.&lt;/video&gt;</td>
    <td>
-    <video controls src="https://archive.org/download/WebmVp8Vorbis/webmvp8_512kb.mp4">The <code>video</code> element is unsupported.</video>
+    <div class="hidden" id="sampleVideo">
+      <pre class="brush: html">&lt;video controls src="https://interactive-examples.mdn.mozilla.net/media/cc0-videos/flower.webm"&gt;The &lt;code&gt;video&lt;/code&gt; element is unsupported.&lt;/video&gt;</pre>
```

Comment:
```suggestion
      <pre class="brush: html">&lt;video controls src="https://interactive-examples.mdn.mozilla.net/media/cc0-videos/flower.webm" style="width: 100%;" &gt;The &lt;code&gt;video&lt;/code&gt; element is unsupported.&lt;/video&gt;</pre>
```
that does the trick for me. Not perfect, but better than a dead link.

Btw the audio examples also seem to not work for me. Looks like this site needs some love in general